### PR TITLE
Query everything thru DBAL connection (to trigger DBAL hooks)

### PR DESCRIPTION
**Pull request by mvorisek**
here is example how to hook for all executed queries:
https://github.com/atk4/data/blob/5a67e085474005c8218b345b224e8f1bb2d94098/src-schema/PhpunitTestCase.php#L42-L53

aka replace ConnectionDumper or how it was called...

### CODE REVIEW

**Review by georgehristov — APPROVED** [review]
LGTM

File: src/Expression.php [hunk 1]
```diff
@@ -522,8 +523,6 @@ public function execute(object $connection = null)
 
         // If it's a PDO connection, we're cool
         if ($connection instanceof DbalConnection) {
-            $connection = $connection->getWrappedConnection();
```
Comment (mvorisek) [inline src/Expression.php:525]: this is basically the one and only MAJOR change

**Comment by georgehristov** [comment]
Working on running everything on DBAL, keeping only Expression class as ATK specific and very good feature

**Comment by mvorisek** [comment]
Yes, dropping DSQL/Connection mess & specific DB vendor implementations is highly welcomed, then DSQL/Expression and DSQL/Query (too) should probably stay for the first PRs.

I think, there should be two PRs - one for dropping all DSQL DB vendor classes, one for dropping the actual DSQL Connection class then.

**Pull request merged**